src/main.h

```diff
@@ -341,7 +341,22 @@ bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime);
  */
 bool CheckFinalTx(const CTransaction &tx, int flags = -1);
```

Comment:
Like the symmetry, but the name here is now a bit out-dated and possibly misleading since we have two types of finality.